### PR TITLE
add local dev and update docs for Sling

### PR DESCRIPTION
**Pull request by cnolanminich**
the Sling loader was failing in prod, and I wanted to debug it. There must have been an update to Sling between version upgrades because we needed to change the AWS_BUCKET from `s3://{bucket_name}/{folder_path}` (old) to just `{bucket_name}`. That was an env var change which is now live, but for me to figure that out I had to get the local version working. This PR adds that local support:

* updates the extras in `setup.py` to include `dagster-embedded-elt`
* adds a local duckdb target and makes local dev supportable
* updates the README.md

### CODE REVIEW

**Review by slopp — APPROVED** [review]
lgtm thanks for getting this all sorted

File: README.md [hunk 1]
```diff
@@ -30,6 +30,14 @@ To understand the structure, start with the file `hooli_data_eng/definitions.py`
...
 - *Alerts* are enabled through Dagster Clould alert policies based on job tags. A custom alert is also specified to notify when assets with SLAs are later than expected. See `hooli_data_eng/assets/delayed_asset_alerts.py`. 
 
+## Assets spanning Multiple Code Locations
+
+- to run the `locations` dataset, uncomment out line 5-7 in [workspaces.yml](workspaces.yml)
+- you will also need to install the packages in the "sling" extra (e.g. `pip install -e ".[dev,sling]")`)
+- You'll need to obtain the credentials for AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY
+- then run `dagster dev` and both `hooli_demo_assets` and `hooli_data_eng` code locations will load
+- NOTE: if you are running Sling locally, there is currently an error message if you already have a duckdb database set up the first time you run the Sling integration. You'll need to delete it and have the `location` asset be the the first thing you materialize.
```
Comment (izzye84) [inline README.md:39]: Did you happen to create a bug for this?  I vaguely remember running into it way back when.

Comment (cnolanminich) [reply README.md:39]: You mean on Sling's repo or here for us? Either way I didn't but can do

Comment (izzye84) [reply README.md:39]: I think we should raise it internally and then we can decide if it should go back to Sling too.

Comment (cnolanminich) [reply README.md:39]: done https://github.com/dagster-io/hooli-data-eng-pipelines/issues/79

**Pull request merged**